ui/src/timeMachine/actions/index.ts

```diff
@@ -642,3 +649,20 @@ export const removeCheckThreshold = (level: CheckStatusLevel) => ({
   type: 'REMOVE_CHECK_THRESHOLD' as 'REMOVE_CHECK_THRESHOLD',
   payload: {level},
 })
+
+export const loadNewVEO = (fromContextID: string) => (
+  dispatch: Dispatch<Action>,
+  getState: GetState
+): void => {
+  dispatch(
+    setActiveTimeMachine('veo', {view: createView<XYViewProperties>('xy')})
+  )
+
+  const values =
+    get(getState(), `variables.values.${fromContextID}.values`) || {}
```

Comment:
stylistic suggestion, take it or leave it as you see fit, just putting it out there  :)

[`get`](https://lodash.com/docs/4.17.15#get) lets you specify a default value as its third parameter:

```suggestion
    get(getState(), `variables.values.${fromContextID}.values`, {})
```

Reply:
thanks!